main.py

```diff
@@ -142,6 +153,46 @@ def create_circuit(self):
             result = 'The request body mimetype is not application/json.'
             log.debug('create_circuit result %s %s', result, 415)
             raise UnsupportedMediaType(result)
+        else:
+            log.info('############ validate spec: ################')
```

Comment:
Can you please review this logging? Keep in mind the following good practices when logging: 1) try to keep the log message 'grep-able' and self-contained, which means that we can use tools like grep to parse and extract information; 2) use the proper logging level (e.g., if the message is for debugging purpose, you should use `log.debug` instead of `log.info`).